lib/livebook_web/live/session_live.ex

```diff
@@ -2070,7 +2070,7 @@ defmodule LivebookWeb.SessionLive do
         :markdown -> LivebookWeb.Output.MarkdownComponent
       end
 
-    send_update(module, id: "outputs-#{idx}-output", event: {:append, output.text})
+    send_update(module, id: "outputs-#{idx}-output", event: {:append, output})
```

Comment:
I pushed the output to the component so they have more flexibility in handling each case without duplication.

Reply:
Sounds good! We actually deconstruct each output into attributes for all other components, but I don't think there was a specific reason for that.